site/docs/providers/cometapi.md

```
@@ -0,0 +1,135 @@
+```bash
+curl -H "Authorization: Bearer $COMETAPI_KEY" https://api.cometapi.com/v1/models
+```
+### Method 2: Accessing via the [CometAPI Pricing Page](https://api.cometapi.com/pricing)
+
+Note: The CometAPI models list includes non-chat models. Promptfoo filters obvious non-chat types using ignore patterns to make selection easier.
+
+## Environment Variables
+
+| Variable | Description |
+| --- | --- |
+| `COMETAPI_KEY` | Your CometAPI key. Get one at https://api.cometapi.com/console/token |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)